schema.graphql

```diff
@@ -261,6 +261,82 @@ type FpmmTransaction @entity {
   transactionHash: Bytes!
 }
 
+type User @entity {
```

Comment:
Rename to GelatoUser?

Reply:
renamed to `GelatoUser` to make things clearer.